src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1086,14 +1108,15 @@ int PgScrubber::build_replica_map_chunk()
 
   switch (ret) {
 
-    case -EINPROGRESS:
+    case -EINPROGRESS: {
       // must wait for the backend to finish. No external event source.
       // (note: previous version used low priority here. Now switched to using
       // the priority of the original message)
       m_osds->queue_for_rep_scrub_resched(m_pg,
 					  m_replica_request_priority,
 					  m_flags.priority,
 					  m_current_token);
+      }
```

Comment:
This formatting change is irrelevant to this commit.  Either remove it or separate it into its own commit.